include/gmock-win32.h

```diff
@@ -960,19 +960,21 @@ void patch_module_func_(void** old_fn, TFunc func, TStub stub) {
             func, reinterpret_cast< void* >(stub), old_fn);
 }
 
```

Comment:
Found it, here it is.
Looks good, I was going to make it myself but didn't have time. Thank you!

Reply:
Thanks for your contribution in identifying the issue.